homeassistant/components/sensor/min_max.py

```diff
@@ -150,10 +152,38 @@ def async_update(self):
         """Get the latest data and updates the states."""
         sensor_values = [self.states[k] for k in self._entity_ids
                          if k in self.states]
-        if len(sensor_values) == self.count_sensors:
-            self.min_value = min(sensor_values)
-            self.max_value = max(sensor_values)
-            self.mean = round(sum(sensor_values) / self.count_sensors,
-                              self._round_digits)
-        else:
-            self.min_value = self.max_value = self.mean = STATE_UNKNOWN
+        self.min_value = self.calc_min(sensor_values)
+        self.max_value = self.calc_max(sensor_values)
+        self.mean = self.calc_mean(sensor_values)
+
+    # pylint: disable=R0201
```

Comment:
Any hint, how to? I tried (including google search) but got no solution that worked with lint.